### PR TITLE
[PLAYER-3645] Adding playback speed UI

### PR DESCRIPTION
**Pull request by jpfranco**
This is the first version of the Playback Speed UI. 

Included in this PR:

- Control bar button
- Playback Speed options Popover/Fullscreen menu
- Accessibility
- MoreOptions button
- Basic skin.json config

Still pending:

- Some UI design updates
- Handling edge cases (disabling for 360 video, MyAds interaction, etc.)
- Refactoring the Video Quality panel in order to use same menu component/new styles
- Some cleanup
- **Unit tests** (will add these in a future PR)

### CODE REVIEW

File: js/components/menuPanelItem.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const React = require('react');
+const AccessibleButton = require('./accessibleButton');
+const Icon = require('./icon');
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const CONSTANTS = require('../constants/constants');
+
+const MenuPanelItem = ({
```
Comment (aeng7) [inline js/components/menuPanelItem.js]: Can we include some documentation/jsdocs on how this will be used and how to use this? We should probably consider including jsdocs for parent level items (classes mainly, or in this case the MenuPanelItem const) in newly created files to help with maintenance.

File: js/components/moreOptionsPanel.js [hunk 2]
```diff
@@ -38,113 +40,95 @@ var MoreOptionsPanel = createReactClass({
...
-  },
-
-  removeHighlight: function(evt) {
-    var iconElement = Utils.getEventIconElement(evt);
-    if (iconElement) {
-      var color = this.props.skinConfig.moreOptionsScreen.iconStyle.inactive.color;
-      var opacity = this.props.skinConfig.moreOptionsScreen.iconStyle.inactive.opacity;
-      Utils.removeHighlight(iconElement, opacity, color);
-    }
+  handlePlaybackSpeedClick: function() {
```
Comment (aeng7) [inline js/components/moreOptionsPanel.js]: Can we include a jsdoc for this?

File: js/components/menuPanelItem.js (hunk 1 above)
Comment (jpfranco) [reply js/components/menuPanelItem.js]: Sounds good, will add

File: js/components/moreOptionsPanel.js (hunk 2 above)
Comment (jpfranco) [reply js/components/moreOptionsPanel.js]: Will add

File: js/components/playbackSpeedPanel.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    } = this.props;
+
+    const selectedValue = Utils.getPropertyValue(
+      controller,
+      'state.playbackSpeedOptions.currentSpeed',
+      CONSTANTS.UI.DEFAULT_PLAYBACK_SPEED
+    );
+    const title = Utils.getLocalizedString(
+      language,
+      isPopover ? CONSTANTS.SKIN_TEXT.PLAYBACK_SPEED : '',
```
[the remark this answers is not in the document]
Comment (jpfranco) [reply js/components/playbackSpeedPanel.js]: The fullscreen menu gets its title from the parent component. It's a bit of overkill to get the localized string for an empty string, but I think it's a bit cleaner this way

**Comment by jpfranco** [comment]
@aeng7 is it ok if I add the comments on my next PR? I'm still working on some updates and fixes

File: scss/components/_playback-speed-panel.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+.oo-playback-speed-panel {
```
Comment (aeng7) [inline scss/components/_playback-speed-panel.scss]: Was it intentional for this class to not apply any styles?

**Comment by jpfranco** [comment]
@aeng7 that class is a placeholder, will remove it if it's not used in the next update. I'll make  a note of these comments but I think it it's more efficient for me to include the changes in the next PR instead of switching to this branch

File: tests/components/controlBar-test.js [hunk 5]
```diff
@@ -44,6 +44,7 @@ describe('ControlBar', function() {
         },
         closedCaptionOptions: {},
         multiAudioOptions: {},
+        playbackSpeedOptions: { currentSpeed: 1 },
```
[the remark this answers is not in the document]
Comment (jpfranco) [reply tests/components/controlBar-test.js]: I think for test files it's better to favor compact code, but I can update these if you want

**Comment by aeng7** [comment]
@jpfranco sounds good

File: tests/components/controlBar-test.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (aeng7) [reply tests/components/controlBar-test.js]: Ah, we can leave them as such in the test files then
